<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogArcDigitEdit</class>
 <widget class="QDialog" name="DialogArcDigitEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Arc Digit</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="spBox_StartDigit">
          <property name="maximum">
           <number>359</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="spBox_EndDigit">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>359</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>End</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSpinBox" name="spBox_Radius">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Radius</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Direction</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="rButton_CounerClockwise">
        <property name="text">
         <string>Couner Clockwise</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rButton_Clockwise">
        <property name="text">
         <string>Clockwise</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogArcDigitEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogArcDigitEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spBox_StartDigit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DialogArcDigitEdit</receiver>
   <slot>arcDigitStartChadged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spBox_EndDigit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DialogArcDigitEdit</receiver>
   <slot>arcDigitEndChadged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spBox_Radius</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DialogArcDigitEdit</receiver>
   <slot>arcDigitRadiusChadged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rButton_CounerClockwise</sender>
   <signal>clicked(bool)</signal>
   <receiver>DialogArcDigitEdit</receiver>
   <slot>arcDigitDirectionCounterClockwiseClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rButton_Clockwise</sender>
   <signal>clicked(bool)</signal>
   <receiver>DialogArcDigitEdit</receiver>
   <slot>arcDigitDirectionClockwiseClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>arcDigitStartChadged(int)</slot>
  <slot>arcDigitEndChadged(int)</slot>
  <slot>arcDigitRadiusChadged(int)</slot>
  <slot>arcDigitDirectionCounterClockwiseClicked(bool)</slot>
  <slot>arcDigitDirectionClockwiseClicked(bool)</slot>
 </slots>
</ui>
